### DISCUSSION TITLE
What's Github Action? 🤔

### DISCUSSION THREAD

**Discussion by jomefavourite** [EddieHubCommunity/support#150] [General] [answered]
So I've been seeing github Action alot lately, what is it all about really?

**Comment by eddiejaoude** [comment, marked as answer]
Great question @jomefavourite 

GitHub Action is like Continuous Integration (CI) - some examples include TravisCI, CircleCI. They are for running automation, like: Linters, automated testing, and then deployment etc

In a nutshell, any commands you run locally, you can run on Github Actions so that it happens for every person - there are events you can hook into, like creating an issue.

I created a video on it, that just got released https://www.youtube.com/watch?v=5ncuRFwnrdc&feature=youtu.be
